### ISSUE TITLE
add support for github sponsors

### ISSUE THREAD

**Issue by fkromer** [protontypes/libreselery#168]
Would be nice beeing able to payout project funding money collected with https://github.com/sponsors

**Comment by Ly0n** [comment]
Yes, to improve Github Sponsors is one of the core ideas of the project. We already integrated it in our "Sponsor Button". Do you see other possibilities to improve that?

**Comment by fkromer** [comment]
As far as I know gh sponsors allows to assign a money pool to a project and thought this could be a potential input source.

**Comment by Ly0n** [comment]
Not yet. A larger discussion is going on about that:
https://vriad.com/essays/a-new-funding-model-for-open-source-software

Check also the HN reaction about that:
https://news.ycombinator.com/item?id=23981563

Many aspects the people are asking are actually covered by LibreSelery
